packages/documentation/src/stories/misc/migration-guide/migrationv6-7.component.ts

```diff
@@ -178,9 +174,9 @@ export class MigrationV67Component extends LitElement {
                     <span class="tag tag-sm tag-danger">breaking</span>
                     <p class="info">
                       Use
-                      <a href="/?path=/docs/1b1ea384-7421-4064-ad34-e3f48a36b39f--docs">tags</a>
+                      <a href="/?path=/docs/1b1ea384-7421-4064-ad34-e3f48a36b39f--docs">Tag component documentation</a>
                       to display states, properties, or other metadata. Opt for
-                      <a href="/?path=/docs/12576d97-52c3-49ec-be7b-6d37728b75f5--docs">chips</a>
+                      <a href="/?path=/docs/12576d97-52c3-49ec-be7b-6d37728b75f5--docs">Chip component documentation</a>
                       when presenting dismissible or selectable information.
```

Comment:
Same for these sentences: "Use Tag component documentation to display states, properties, or other metadata. Opt for Chip component documentation when presenting dismissible or selectable information."